spec.bs

```diff
@@ -2444,6 +2444,35 @@ algorithms to achieve the outcomes described in the above explanatory content.
   /fenced-frame/permission-notification.https.html
 </wpt>
 
+<h3 id=cssom-monkeypatch>CSSOM View</h3>
+
+The [[!CSSOM-VIEW]] specification defines scroll bubbling behavior through the [=scroll a target
+into view=] algorithm. Our specification modifies it to not allow scrolling a target into view if it
+is inside of a fenced frame. This is done because the {{Element/scrollIntoView()}} method is an
+event that can both be invoked from and observed by script, which can open a communication side
+channel between a <{fencedframe}> and its embedder.
+
+<div algorithm=scroll-target-into-view>
+  Modify the [=scroll a target into view=] algorithm to add 2 steps after step 1 that read:
+
+  2. If |target|'s associated {{Document}}'s [=node navigable=]'s [=navigable/traversable
+     navigable=] is a [=fenced navigable container/fenced navigable=], terminate these steps.
+
+  3. If |target|'s associated {{Document}}'s [=Document/ancestor navigables=] [list/contains=] a
+     [=navigable=] whose [=navigable/traversable navigable=] is a [=fenced navigable
+     container/fenced navigable=], terminate these steps.
+</div>
+
+<div algorithm=run-the-scroll-steps>
+  Modify the [=Document/run the scroll steps=] algorithm to add a check before step 1.1 that reads:
+
+  1. If <var ignore>target</var> is a {{Document}} whose [=node navigable=]'s
+     [=navigable/traversable navigable=] is a [=fenced navigable container/fenced navigable=],
+     [=iteration/continue=].
+</div>
+
+Note: This ensures that scroll events do not affect fenced frames.
```

Comment:
Please elaborate on this as I don't understand what it means. Does this PR make *all* scroll events impossible to trigger in a fenced frame? That seems wrong, since we don't want to impact any that I guess originated from the other side of the fence, right?

Reply:
Resolving since this is no longer in the patch.